src/script/connection/ConnectionRepository.ts

```diff
@@ -229,7 +234,6 @@ export class ConnectionRepository {
             break;
           }
         }
-        return false;
       }
       throw error;
```

Comment:
Instead of returning false, we just throw an error we will ignore later

Reply:
humm this actually changes the behavior. Previously we would catch unexpected errors, but now they are going to be swallowed into the void